src/templates/mht_part.tpl

```diff
@@ -0,0 +1,6 @@
+------=_NextPart_000_0000_01D0CFF2.897FE540
```

Comment:
It's the name of the content part, it has no meaning. Can be any, e.g 'htmlDocumentPart'. The only important thing is that all parts have same partName string otherwise MHT doc will not be valid.